scripts/stealthy.js

```diff
@@ -46,24 +46,34 @@ export class Stealthy {
           // look for effects that indicate Dim or Dark condition on the token
           const  tokenLight = game.settings.get('stealthy', 'tokenLighting');
           if (tokenLight){
-            let lightLevel = 2; // bright
+            let lightLevel = 0;
```

Comment:
default light level is dark instead of bright?

Reply:
thanks